### PR TITLE
Add Pavuk Package

### PR DESCRIPTION
**Pull request by beliefless-dogmasless**
Hy,

It works(compile) well on arm.

### CODE REVIEW

File: packages/pavuk.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Pavuk < Package
+  description 'Web spider and website mirroring tool. http://www.pavuk.org/'
+  homepage 'https://sourceforge.net/projects/pavuk/files/pavuk/'
```
Comment (uberhacker) [inline packages/pavuk.rb]: Please do not include the homepage url in the description.  If you want to propose that we add a new field to the Package class, then please create a new issue so we can have a discussion.  The homepage url you have there also points at the source, not the homepage of the project.

**Comment by uberhacker** [comment]
Thank you for the new package @beliefless-dogmasless.  Please see inline comment.

File: packages/pavuk.rb (hunk 1 above)
Comment (beliefless-dogmasless) [reply packages/pavuk.rb]: I don't want to 'spend' 'more time' on talking, I will contribute, but remove one link(in the section, the direct link to download). Do, what you want with rest I'm not interested(don't take it personally, it is not).

**Comment by uberhacker** [comment]
If you need the homepage and version of the package, you can execute `crew search <package> -v`.

**Pull request merged**